lib/mutant/mutator/node/send.rb

```diff
@@ -26,8 +26,9 @@ class Send < self
           to_i:          %i[to_int],
           to_a:          %i[to_ary],
           to_h:          %i[to_hash],
-          at:            %i[fetch],
-          :[] =>         %i[at fetch],
+          at:            %i[fetch key?],
+          fetch:         %i[key?],
+          :[] =>         %i[at fetch key?],
```

Comment:
@dkubb You could even maybe just define the mapping in terms of one method name to another method name like so

```ruby
{
  reverse_map:   :map,
  map:           :each,
  kind_of?:      :instance_of?,
  is_a?:         :instance_of,
  reverse_each:  :each,
  reverse_merge: :merge,
  send:          :__send__,
  __send__:      :public_send,
  gsub:          :sub,
  eql?:          :equal?,
  to_s:          :to_str,
  to_i:          :to_int,
  to_a:          :to_ary,
  to_h:          :to_hash,
  at:            :fetch,
  fetch:         :key?,
  :[] =>         :at,
  :== =>         :eql?,
  :>= =>         :>,
  :>  =>         :==,
  :<= =>         :<,
  :<  =>         :==
}
```

with that you can then infer that a set of mappings like this

```ruby
  :[] =>         :at,
  at:            :fetch,
  fetch:         :key?,
```

means that `:[]` is semantically simplified to `:at`, `:at` to `:fetch`, and `:fetch` to `:key?`.